src/Data/Barbie/Internal/Constraints.hs

```diff
@@ -109,6 +111,14 @@ bmapC f bf = bmap go (baddDicts bf)
     go :: forall a. (Dict c `Product` f) a -> g a
     go (d `Pair` fa) = requiringDict (f fa) d
 
+-- | Like 'btraverse' but with a constraint all elements of @b@.
```

Comment:
how about "`a constraint on the elements of @b@`"?

Reply:
Classic mistake on my behalf: Amend the commit but don't add the actual changes ...